libs/core/src/lib/avatar-group/avatar-group.component.scss

```diff
@@ -767,3 +767,14 @@
     color: #fff;
     color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
 }
+
+// Keep it here after removing styles
+.fd-avatar-group--group-type {
+    fd-popover-control:focus,
+    .fd-popover__control:focus {
+        outline-offset: 0.0625rem;
+        outline-width: 0.0625rem;
+        outline-color: #000;
```

Comment:
why can't we have this in fundamental styles? and why do you use hardcoded color?

Reply:
the 3 values are hard-coded. It should be:

```
outline-width: var(--sapContent_FocusWidth);
outline-color: var(--sapContent_FocusColor);
outline-style: var(--sapContent_FocusStyle);
```

Reply:
but it should also go in Fundamental-styles

Reply:
@droshev @InnaAtanasova thanks, I'll shortly raise an issue in the styles repository to fix it there. 
Now fixed here.